examples/systemcalls/systemcalls.c

```diff
@@ -93,6 +126,43 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
+	pid_t child_pid;
```

Comment:
@akshithaluguri22 most of this function is a duplicate of the function above.  Think about a way you could design in a way which doesn't require duplication of code.